quickwit/quickwit-control-plane/src/model/mod.rs

```diff
@@ -392,12 +402,157 @@ impl ControlPlaneModel {
         self.shard_table
             .release_scaling_permits(source_uid, scaling_mode)
     }
+
+    // Quickwit 0.9 uses the ingest v2 source by default. For indexes created prior to 0.9, we need
+    // to ensure that the ingest v2 source is created and enabled if necessary.
```

Comment:
I think this deserves a few lines in an [upgrade docs page](https://quickwit.io/docs/main-branch/operating/upgrades)